app/assets/stylesheets/govuk_publishing_components/components/_inverse-header.scss

```diff
@@ -1,5 +1,15 @@
-.gem-c-inverse-header {
+%inverse-header {
```

Comment:
If you follow BEM you can avoid generating additional CSS.

```css
.inverse-header { ... } /* default styles */
.gem-c-inverse-header--full-width {
  padding-left: 0;
  padding-right: 0;
}
```

```erb
<%
  full_width ||= false
  header_class = full_width ? "gem-c-inverse-header--full-width" : ""
%>
<% block = yield %>
<% unless block.empty? %>
  <header class="gem-c-inverse-header <%= header_class %>">
    <%= block %>
  </header>
<% end %>
```

Reply:
Thanks @nickcolley! That is really useful.